cluster-sync/clean.sh

```diff
@@ -7,7 +7,7 @@ source ./cluster-up/cluster/${KUBEVIRT_PROVIDER}/provider.sh
 echo "Cleaning up ..."
 
 if [ "${CDI_CLEAN}" == "test-infra" ]; then
-  _kubectl delete all -n cdi -l cdi.kubevirt.io/testing
+  _kubectl delete all -n ${CDI_NAMESPACE} -l cdi.kubevirt.io/testing
```

Comment:
nice catch

Reply:
Yeah, apparently there are some more resources we want to get rid of that are not namespaced..
scc for example, which doesn't show up in "get all"